.github/actions/build-gk/action.yaml

```diff
@@ -31,15 +31,15 @@ runs:
     - name: install mamba
       uses: ./.github/actions/install-mamba
       with:
-        installer-url: https://micro.mamba.pm/api/micromamba/${{ inputs.platform }}/1.5.7
+        installer-url: https://micro.mamba.pm/api/micromamba/${{ inputs.platform }}/1.5.8
 
     - name: create build env
       shell: bash -l -e {0}
       run: |
         set -x
         if [ ! -d "${HOME}/micromamba/envs/build" ]; then
           micromamba create -yqn build -c conda-forge \
-            boa==0.16.0 mamba==1.5.7 conda==24.1.2 \
+            boa==0.17.0 mamba==1.5.8 conda=24.5 \
```

Comment:
weird, I remember trying the same combination (but removed mamba) and it not working.